mounter/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package mounter defines an interface to mounting filesystems.
+package mounter // import "k8s.io/utils/mounter"
```

Comment:
Yes. we had chose `mounter` to avoid having to rename `MountPoint`, but I went ahead and did it now...

https://github.com/kubernetes/kubernetes/pull/81186